docs/guides/guide-data-sovereignty-and-protocols/index.md

```diff
@@ -0,0 +1,66 @@
+
+IDSov challenges the historical backdrop of data exploitation where Indigenous data was often extracted and used without consent, resulting in its misappropriation and devaluation. IDSov strives to counteract the colonial legacy of resource and knowledge extraction, ensuring that Indigenous Peoples control how data about their communities is collected, used, and shared. For example, Indigenous data sovereignty principles diverge sharply from conventional intellectual property and data privacy laws, as they are rooted in a relational worldview that prioritizes interconnectedness among people, land, and the cosmos. Knowledge protocols from Indigenous communities emphasize this interconnectedness, describing their protocols as “living beings” that evolve to reflect the continuous journey of the community’s identity and heritage.
+
+### What are the CARE Principles for Indigenous Data Governance?
+
+The CARE Principles for Indigenous Data Governance are a set of principles that guide the collection, use, and sharing of Indigenous data. Influenced by First Nations Principles of OCAP (Ownership, Control, Access and Possession) in Canada, the Principles of Māori Data Sovereignty, and more, they attempt to articulate a comprehensive and global framework for Indigenous data governance. The CARE Principles are:
+
+* **[C]ollective Benefit** - Data ecosystems must enable Indigenous Peoples to derive benefit from the data
+
+* **[A]uthority to Control** - Indigenous Peoples' rights and interests in Indigenous data must be recognized and their authority empowered
+
+* **[R]esponsibility** - Those working with Indigenous data must share how data are used to support self-determination and collective benefit
+
+* **[E]thics** - Indigenous Peoples' rights and wellbeing should be the primary concern at all stages of the data lifecycle
+
+Each principle is supported by a set of sub-principles that describe how the principle should be applied in practice. For more information, see "Care Principles for Indigenous Data Governance" (2020)[^2].
+
+The CARE Principles are an important standard that is already being used to benchmark and improve data sharing and stewardship practices. For example, Jennings et al. (2023)[^3] used the CARE Principles to recommend better data stewardship practices for open earth science research.
```

Comment:
Note: later, we can refer to this guide to benchmark whether and how Guardian Connector evaluates against e.g. the CARE principles!